app/src/main/java/org/systers/mentorship/view/fragments/MembersFragment.kt

```diff
@@ -127,6 +131,23 @@ class MembersFragment : BaseFragment() {
                                 isRecyclerView = true
                             }
                         }
+                        else
+                        {
+                            if(filterMap.containsKey("location")&&filterMap["location"] != "")
+                            {
+                                val User_list = membersViewModel.userList
+                                val query = filterMap["location"].toString()
+
+                                val location_check = User_list.any {(it.location)?.contains(query, ignoreCase = true) == true }
```

Comment:
Ik proguard would handle this but you dont really need this variable, just inline it.

Reply:
So you are asking to inline these variable without creating them separately?

Reply:
Yeah